pkg/system/phase4_configuring.go

```diff
@@ -318,6 +318,8 @@ func (r *Reconciler) SetDesiredDeploymentEndpoint() error {
 					if r.JoinSecret == nil {
 						c.Env[j].Value = r.MongoConnectionString
 					}
+				case "NOOBAA_LOG_LEVEL":
+					c.Env[j].Value = strconv.Itoa(r.NooBaa.Spec.DebugLevel)
```

Comment:
but what if the spec doesn't include DebugLevel?

Reply:
you can't rely on the CLI setting it because the NooBaa CR can be created by yaml...

Reply:
Oh but perhaps this is OK because the variable is int, and when empty it is omited, but still returned as 0 in golang when empty.